### PR TITLE
Make upload text files like html

### PR DESCRIPTION
**Pull request by rusko124**
When i try upload html page, i had error message like "Bad file format".

### CODE REVIEW

**Comment by vpetersson** [comment]
I don't know if we want to support this to be honest. It's not going to work well in most cases, so I think it's better to throw an error.

**Comment by vpetersson** [comment]
@rusko124 This PR doesn't seem to do what it describes. Also, see my comment.

**Comment by rusko124** [comment]
@vpetersson i made mistake this changes for https://github.com/Screenly/screenly-ose/pull/915.

File: server.py [hunk 1]
```diff
@@ -819,8 +819,7 @@ def post(self):
         req = Request(request.environ)
         file_upload = req.files.get('file_upload')
         filename = file_upload.filename
-
-        if guess_type(filename)[0].split('/')[0] not in ['image', 'video']:
+        if guess_type(filename)[0].split('/')[0] not in ['image', 'video', 'text']:
```
Comment (vpetersson) [inline server.py]: I don't like this. Let's remove the text.

**Pull request closed without merging**